devices/models.py

```diff
@@ -100,6 +102,35 @@ class Bookmark(models.Model):
     user = models.ForeignKey(Lageruser)
 
 
+class Vendor(models.Model):
+    name = models.CharField(_('Name'), max_length=100)
+    driver_url = models.CharField(_('Drivers'), max_length=1000, blank=True, null=True)
+    support_url = models.CharField(_('Support'), max_length=1000, blank=True, null=True)
```

Comment:
Also, I was thinking about a more generic approach to support URLs. I mean, they could be totally different for each vendor. A number of generic URL templates would be much more flexible. This way a user could add a support and a drivers URL with a URL template for Dell and a single selfservice URL template for Apple.